pennylane/qchem/hamiltonian.py

```diff
@@ -340,7 +341,7 @@ def molecular_hamiltonian(*args, **kwargs):
 
             ``molecular_hamiltonian``\\ (`symbols, coordinates, name='molecule', charge=0, mult=1, basis='sto-3g',`
             `method='dhf', active_electrons=None, active_orbitals=None, mapping='jordan_wigner', outpath='.',`
-            `wires=None, alpha=None, coeff=None, args=None, load_data=False, convert_tol=1e12`)
+            `wires=None, coeff=None, alpha=None, args=None, load_data=False, convert_tol=1e12`)
```

Comment:
I'm changing the order here just so it's a bit more convenient in the generate_params function. If this is an issue I can always revert it but it seems to be fine

Reply:
Why this is an issue with generate_params? It is proffered to keep the UI the same as much as we can. But feel free to change the order in private functions if it helps.

Reply:
It's ok, no longer an issue so I reverted back to the old UI.